great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -225,11 +232,22 @@ def _render_other(  # noqa: PLR0912, PLR0913
     ) -> Any:
         """Helper method to render non-list render_objects - refer to `render` for more context"""
         expectation_type = cls._get_expectation_type(render_object)
+        expectation_config = (
+            render_object.expectation_config
+            if isinstance(render_object, ExpectationValidationResult)
+            else render_object
+        )
 
-        content_block_fn = get_renderer_impl(
-            object_name=expectation_type, renderer_type=LegacyRendererType.PRESCRIPTIVE
+        content_block_fn = cls._get_content_block_fn(
+            expectation_type=expectation_type, expectation_config=expectation_config
         )
-        content_block_fn = content_block_fn[1] if content_block_fn else None
+        if not content_block_fn:
+            renderer_impl = get_renderer_impl(
+                object_name=expectation_type,
+                renderer_type=LegacyRendererType.PRESCRIPTIVE,
+            )
+            content_block_fn = renderer_impl[1] if renderer_impl else None
```

Comment:
Quick rename since `content_block_fn` should be a `Callable` rather than a `tuple`